src/Functions/array/arrayResize.cpp

```diff
@@ -21,117 +21,99 @@ namespace ErrorCodes
     extern const int ILLEGAL_TYPE_OF_ARGUMENT;
 }
 
-class FunctionArrayResize : public IFunction
+DataTypePtr FunctionArrayResize::getReturnTypeImpl(const DataTypes & arguments) const
```

Comment:
The diff looks messy, but only indentation was changed, and part of the code was moved into .h